cea/interfaces/dashboard/api/project.py

```diff
@@ -102,94 +104,104 @@ class Scenarios(Resource):
+                    locator.ensure_parent_folder_exists(locator.get_building_age())
+                    locator.ensure_parent_folder_exists(locator.get_building_occupancy())
+                    locator.ensure_parent_folder_exists(locator.get_street_network())
+
+                    if 'zone' in files:
+                        for filename in glob_shapefile_auxilaries(files['zone']):
+                            shutil.copy(filename, locator.get_building_geometry_folder())
+                    if 'surroundings' in files:
+                        for filename in glob_shapefile_auxilaries(files['surroundings']):
+                            shutil.copy(filename, locator.get_building_geometry_folder())
+                    if 'surroundings' in files:
+                        for filename in glob_shapefile_auxilaries(files['surroundings']):
+                            shutil.copy(filename, locator.get_building_geometry_folder())
+                    if 'terrain' in files:
+                        shutil.copyfile(files['terrain'], locator.get_terrain())
+                    if 'streets' in files:
+                        shutil.copyfile(files['streets'], locator.get_street_network())
+
+                    from cea.datamanagement.zone_helper import calculate_age_file, calculate_occupancy_file
+                    if 'age' in files and files['age'] != '':
+                        shutil.copyfile(files['age'], locator.get_building_age())
+                    elif 'zone' in files:
+                        zone_df = geopandas.read_file(files['zone'])
+                        calculate_age_file(zone_df, None, locator.get_building_age())
+
+                    if 'occupancy' in files and files['occupancy'] != '':
+                        shutil.copyfile(files['occupancy'], locator.get_building_occupancy())
+                    elif 'zone' in files:
+                        zone_df = geopandas.read_file(files['zone'])
+                        if 'category' not in zone_df.columns:
+                            # set 'MULTI_RES' as default
+                            calculate_occupancy_file(zone_df, 'MULTI_RES', locator.get_building_occupancy())
+                        else:
+                            calculate_occupancy_file(zone_df, 'Get it from open street maps', locator.get_building_occupancy())
+                except Exception as e:
+                    import traceback
+                    trace = traceback.format_exc()
+                    return {'message': e.message, 'trace': trace}, 500
 
         elif payload['input-data'] == 'copy':
-            source_scenario = os.path.join(config.project, payload['copy-scenario'])
-            shutil.copytree(cea.inputlocator.InputLocator(source_scenario).get_input_folder(),
-                            locator.get_input_folder())
+            try:
+                os.makedirs(scenario_path)
+                source_scenario = os.path.join(temp_config.project, payload['copy-scenario'])
```

Comment:
this might be where #2508 got introduced...

Reply:
Fixed in bca24d8. Probably something to do with `os.makedirs(scenario_path)`. Did not have this issue when trying to copy a new scenario, only when copying a scenario that already has some outputs from scripts. weird.